test/docker/minio.go

```diff
@@ -58,6 +58,7 @@ func startMinIO(ctx context.Context, networkName string) (*DockerContainer, erro
 	}
 	envSettings := make(map[string]string)
 	envSettings["BACKUP_S3_ENDPOINT"] = fmt.Sprintf("%s:%s", MinIO, port.Port())
+	envSettings["S3_ENDPOINT_URL"] = fmt.Sprintf("http://%s:%s", MinIO, port.Port())
```

Comment:
what is the difference between `OFFLOAD_S3_ENDPOINT` and `S3_ENDPOINT_URL`? can we just use `OFFLOAD_S3_ENDPOINT`? this brings a little bit of ambiguity

Reply:
Seems to me that `OFFLOAD_S3_ENDPOINT` is external and user facing, and `S3_ENDPOINT_URL` is the internal naming. Maybe to avoid the redundancy of prefixing it with `OFFLOAD` since it's already an `offload` module?

Not sure, but I'm not opposed to making it more explicit

Reply:
@marcin actually this is probably wrong, I see both [here](https://github.com/weaviate/weaviate/pull/5119/files#diff-ebfe6edecbb1636093cc9e9d0ac10a40007c2b1eba87b83542f37a423e2cd132R524-R525) in the `run_dev_server` script